Algorithm-Factorialize-A-Number.md

```diff
@@ -1,14 +1,18 @@
 # Algorithm Factorialize A Number
 
-## Problem Explanation:
+:triangular_flag_on_post: Remember to use [**`Read-Search-Ask`**](FreeCodeCamp-Get-Help) if you get stuck. Try to pair program :busts_in_silhouette: and write your own code :pencil:
+
+![Recursion](http://i61.tinypic.com/28auvsw.jpg)
+
+## :checkered_flag: Problem Explanation:
 
 Return the factorial of the provided integer. If the integer is represented with the letter n, a factorial is the product of all positive integers less than or equal to n.
 
 Factorials are often represented with the shorthand notation n!
 
 For example: `5! = 1 * 2 * 3 * 4 * 5 = 120`
 
-## Outline
+## Hints
```

Comment:
Change the other to that too, you can add the previous title in bold underneath.